hazelcast-ra/hazelcast-jca/src/test/java/com/hazelcast/jca/HazelcastConnectionImplTest.java

```diff
@@ -191,7 +190,7 @@ public void getJobTracker() {
     @Test
     public void getCluster() {
         Cluster cluster = connection.getCluster();
-        assertNotSame(cluster, hz.getCluster());
```

Comment:
Just a question: what was returned in the old situation? Some kind of protective wrapper?